test/openAPI/features/wallet_generate_link_auth_code.feature

```diff
@@ -4,7 +4,7 @@ Feature: The endpoint to validates the link-code and its expiry and generates th
   @smoke @unit @positive
   Scenario: Successfully validates the link-code and its expiry and generates the link auth code smoke type test
     Given Wants to validate the link-auth-code and generate the auth code
-    And The link code is generated before POST /linked-authorization/link-auth-code
+    And The link code, transaction and authenticate is completed before POST /linked-authorization/link-auth-code
```

Comment:
I can see we are generating link code, linking the transaction and authenticating, but we also need to do the consent (/linked-authorization/consent) before calling link-auth-code.

Moving comment from old PR (https://github.com/GovStackWorkingGroup/bb-identity/pull/34#discussion_r1243118437)